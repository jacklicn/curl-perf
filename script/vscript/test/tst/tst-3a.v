var i,b

i = 0
b = 2
do
   if i % 2 == 0 
     i = i + 1
     continue
   end
   
   print(i)

   i = i + 1
   b = b * 2
until i < 7

print(i)
print(b)

