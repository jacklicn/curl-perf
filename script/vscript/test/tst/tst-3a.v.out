*** syntax ok ***
*** semantics and code generation ok ***

>1

>3

>5

>7

>8

>32
